README.md

```diff
@@ -1,39 +1,65 @@
 [![Coverage Status](https://coveralls.io/repos/github/temporalio/temporal/badge.svg?branch=master)](https://coveralls.io/github/temporalio/temporal?branch=master)
 [![Discourse](https://img.shields.io/static/v1?label=Discourse&message=Get%20Help&color=informational)](https://community.temporal.io)
 
-Visit [docs.temporal.io](https://docs.temporal.io) to learn about Temporal.
+# Temporal  
+
+Temporal is a microservice orchestration platform which enables developers to build scalable applications without sacrificing productivity or reliability.
+Temporal server executes units of application logic, Workflows, in a resilient manner that automatically handles intermittent failures, and retries failed operations.
 
-This repo contains the source code of the Temporal server. To implement workflows, activities and worker use [Go SDK](https://github.com/temporalio/sdk-go) or [Java SDK](https://github.com/temporalio/sdk-java).
+The programming model of Temporal exposed via the SDKs (currently for Java and Go) offers developers the simple abstractions of Workflows and Activities that can virtually "resume" execution after a failure from the same line of code.
 
-See Maxim's talk at [Data@Scale Conference](https://atscaleconference.com/videos/cadence-microservice-architecture-beyond-requestreply) for an architectural overview of Temporal.
+Temporal is a mature technology, a fork of Uber's Cadence.
+Temporal is being developed by [Temporal Technologies](https://temporal.io/), a startup by the creators of Cadence.
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/f-18XztyN6c/0.jpg)](http://www.youtube.com/watch?v=f-18XztyN6c "Temporal")
+
+Learn more about Temporal at [docs.temporal.io](https://docs.temporal.io).
 
 ## Getting Started
 
-### Start the temporal-server locally
+### Download and start Temporal server locally
+
+Execute the following commands to start a pre-built image along with all the dependencies.
 
-We highly recommend that you use [Temporal service docker](docker/README.md) to run the service.
+```bash
+$ curl -L https://github.com/temporalio/temporal/releases/latest/download/docker.tar.gz | tar -xz
+$ cd docker
+$ docker-compose up
+```
+
+Refer to [Temporal service docker](docker/README.md) for more advanced options.
 
 ### Run the Samples
 
-Try out the sample recipes for [Go](https://github.com/temporalio/samples-go) or [Java](https://github.com/temporalio/samples-java) to get started.
+Clone or download samples for [Go](https://github.com/temporalio/samples-go) or [Java](https://github.com/temporalio/samples-java) and try to run them against the local Temporal server.
 
 ### Use CLI
 
-Try out [Temporal command-line tool](tools/cli/README.md) to perform various tasks on Temporal
+Use [Temporal's command line tool](https://docs.temporal.io/docs/tctl) `tctl` to interact with the local Temporal server.
+
+```bash
+$ tctl namespace list
+$ tctl workflow list
```

Comment:
```zsh
alias tctl="docker run --rm temporalio/tctl:1.1.0 --address host.docker.internal:7233"
tctl namespace list
tctl workflow list
```

Reply:
`alias` sounds overwhelming for me. Especially pining it to exact version here. But I agree, we recommend docker-compose few lines above, and it assumes that users won't build anything themselves (let's say they want to play with Java SDK and don't have `go` at all).

Reply:
I set version to "latest".